Code/Framework/AzFramework/AzFramework/DocumentPropertyEditor/ReflectionAdapter.h

```diff
@@ -60,6 +62,10 @@ namespace AZ::DocumentPropertyEditor
         //! property editor instances has altered its value.
         void NotifyPropertyChanged(const PropertyChangeInfo& changeInfo);
 
+        virtual void OnBeginRow(AdapterBuilder*, const AZStd::string_view&)
+        {
+        }
```

Comment:
Repeat the implementation should be moved to the cpp file.
Also the `const AZStd::string_view&` should be `AZStd::string_view`
Finally since this is an interface function, it needs a doxygen comment explaining what it is for.